src/layman/layer/util.py

```diff
@@ -126,12 +126,40 @@ def get_complete_layer_info(workspace, layername, *, x_forwarded_items=None):
                                                      complete_info_method=_get_complete_layer_info)
 
 
+def get_complete_layer_info_by_uuid(*, uuid=None, info=None, x_forwarded_items=None):
```

Comment:
Is this method called with `info` argument somewhere? If not, remove the `info` argument and related logic.